Content.Tests/DMProject/Tests/Preprocessor/EmptyAndDirectiveMacros.dm

```diff
@@ -0,0 +1,26 @@
+﻿#define EMPTY
+#define EMPTY_FN(a)
+#define DEFINE #define
+#define DEFINE_WITH_EMPTY EMPTY #define
+#define DEFINE_NESTED(NAME, VALUE) \
+	DEFINE NAME VALUE
+#define DEFINE_NESTED_WITH_STATEMENT(NAME, VALUE) \
+	var/test = 4\
+	DEFINE NAME VALUE
+
+DEFINE A 1
+DEFINE_WITH_EMPTY B 2
+DEFINE_NESTED(C, 3)
+EMPTY DEFINE D 4
+ DEFINE E 5
```

Comment:
BYOND doesn't like the space at the beginning of this line

Reply:
That seems like a BYOND bug? swapping `DEFINE` for `#define` compiles just fine. I can't really see any impact in making it more permissive.